capstone/src/main/java/com/google/univiz/UniversityData.java

```diff
@@ -0,0 +1,90 @@
+package com.google.univiz;
+
+import com.google.auto.value.AutoValue;
+
+/** UniversityData is a class representing a single university. */
+@AutoValue
+public abstract class UniversityData {
+
+  /**
+   * Creates a new UniversityData
```

Comment:
Overall looking good :) there are a lot of comments already here, so I'll rereview when those comments are resolved and see if I have anything else to add